core/resources/scene.yaml

```diff
@@ -110,8 +122,13 @@ layers:
         draw:
             polygons:
                 order: 1
-                color: '#fffffa'
+                #color: '#fffffa'
+                color: global.color.colorA
                 interactive: true # currently ignored
+            lines:
+                order: 2
+                color: function() { return global.color.colorE; }
```

Comment:
Ok, so whats happening right now is, when creating the global JS object for duk's context, it evaluates any JS function it finds in any of the global properties.

And when evaluating duk context for styleParams or filter it simply replaces the evaluated global value.

Even though `global.prop.propA()` is the correct way of calling a JS function, in our case should the usage be to simply put the evaluated value where ever a global property is accessed?
Also tried in JS, it doesn't seem to work there!